src/Select/index.js

```diff
@@ -184,6 +172,22 @@ class Select extends Component {
         return this.refs.menu;
     }
 
+    getButtonText() {
+        const value = this.props.value;
+        const text = this.getItems().reduce((res, item) => {
+            if (value.indexOf(item.props.value) !== -1) {
+                if (value.length === 1) {
+                    res.push(Component.textValue(item));
+                } else {
+                    res.push(item.props.checkedText || Component.textValue(item));
+                }
+            }
+            return res;
+        }, []);
+
+        return text.join(', ') || this.props.placeholder || '—';
```

Comment:
Yes, but it is easier to override when the base method doesn't know about placeholders.

```
renderButtonText() {
  const text = super.renderButtonText();
  
  return text && `Days: ${text}`;
}
```

Conceptually, seems like it's ok to show placeholders if `renderButtonText` returns nothing.